### PR TITLE
[17.0][MIG] maintenance_project

### PR DESCRIPTION
**Pull request by Anxo82**
Remove in demo_maintenance_project.xml 2 fields: "maintenance_duration" and "period". This fields removed in base odoo.

### CODE REVIEW

File: maintenance_project/test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+odoo-addon-base_maintenance @ git+https://github.com/OCA/maintenance.git@refs/pull/400/head#subdirectory=base_maintenance
```
Comment (dalonsod) [inline maintenance_project/test-requirements.txt]: @Anxo82 test requirements file seems to be good, but it should be placed in the root folder, not inside addon directory.

**Comment by dalonsod** [comment]
@FernandoRomera tests fixed, could you review?

**Review by FernandoRomera — APPROVED** [review]
@dalonsod 
LGTM.
A silly thing, to follow OCA guidelines. Rename dir data to demo

**Comment by dalonsod** [comment]
@mymage could you review? Thanks!

**Comment by dalonsod** [comment]
> A silly thing, to follow OCA guidelines. Rename dir data to demo

@FernandoRomera done!

**Review by mymage — APPROVED** [review]
LGTM
As improvement, but it is not argument of the migration, I'll add equipment reference on Project form and Project reference on Equipment form. I'll do an issue just as reminder.

File: maintenance_project/views/maintenance_equipment_views.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    </record>
+    <record id="hr_equipment_view_form" model="ir.ui.view">
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='location']" position="after">
+                <field name="create_project_from_equipment" invisible="id" />
+                <field
+                    name="project_id"
+                    invisible="create_project_from_equipment or not id"
```
Comment (dalonsod) [inline maintenance_project/views/maintenance_equipment_views.xml]: @Anxo82 according to original code this condition's been translated wrong in new syntax, please check:
```suggestion
                    invisible="create_project_from_equipment and not id"
```

Comment (Anxo82) [reply maintenance_project/views/maintenance_equipment_views.xml]: Done! Review please.
Thanks.

Comment (dalonsod) [reply maintenance_project/views/maintenance_equipment_views.xml]: It works

**Comment by dalonsod** [comment]
@etobella could you review? Thanks!

**Review by etobella — APPROVED** [review]

**Pull request merged**